docs/source/contributing.rst

```diff
@@ -50,17 +50,17 @@ Build the relayers::
 Start ganache::
 
     ganache --wallet.totalAccounts 10 --chain.hardfork london --miner.blockGasLimit 12000000 \
-            --wallet.mnemonic brownie --server.port 8545 --chain.chainId 1337
+            --wallet.hdPath "m/44'/60'/0'" --wallet.mnemonic "always host near cat visual knock inspire multiply there eight weapon fat" --server.port 8545 --chain.chainId 1337
```

Comment:
There has to be a better way for this. :-)
Doesn't `ganache` support an `ape` mnemonic?
How hard would it be to add it if not?

Reply:
Created 2 PR for this: https://github.com/ApeWorX/ape/pull/1299 and https://github.com/ApeWorX/ape-ganache/pull/22

Reply:
Awesome.

BTW,  what about this:
> Doesn't ganache support an ape mnemonic?

Reply:
what does it mean ape mnemonic and ganache support?

Reply:
> what does it mean ape mnemonic and ganache support?

We now have brownie working with ganache out of the box, without us having to provide `--wallet.hdPath` and `--wallet.mnemonic` options to ganache. The question is, how can we continue not specifying these options to ganache and make ape work?

Reply:
brownie is using w3.eth.accounts to create accounts and ape is not doing that and if they do that means they are removing tx signing functionality of all TestAccounts and remaining would be like all accounts becoming ImpersonatedAccounts or so which is not exactly true in ape conditions as ImpresonatedAccounts we dont have any private keys but we have it with test accounts. We discussed this a little also in discord this is not going to happen imo.